docs/reference/search/search.asciidoc

```diff
@@ -558,12 +558,20 @@ Period of time used to extend the life of the PIT.
 (Optional, <<query-dsl,query object>>) Defines the search definition using the
 <<query-dsl,Query DSL>>.
 
+[[request-body-retriever]]
+`retriever`::
+preview:[]
+(Optional, <<retriever, retriever object>>) Defines a top-level retriever to specify
+a desired set of top documents instead of a standard query or knn search.
+
 [[request-body-rank]]
 `rank`::
 preview:[]
 This param is in technical preview and may change in the future. The syntax will
```

Comment:
Should we remove the verbiage about syntax changing prior to GA for these features?

Reply:
I think this is pretty standard until a feature does become GA just in case we discover any design flaws, so I'd prefer to error on the side of caution here.